multi_import/formats.py

```diff
@@ -99,6 +104,8 @@ def write(self, dataset):
 
 
 class CsvFormat(TabLibFileFormat):
+    mime_type = "text/csv"
```

Comment:
Should this be `content_type` instead?

Reply:
No, the content_type is currently `application/csv` whereas the mime type is `text/csv`

I still need to look into what the significance of that is.

Reply:
I did some testing, seems there is little impact if I change content_type from `application/csv` to `text/csv`

I have updated the pull request, to make this change - which has helped to simply things.